mlir/include/mlir/Dialect/MQTDyn/Transforms/Passes.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+ * Copyright (c) 2025 Munich Quantum Software Company GmbH
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+#pragma once
+
+#include <mlir/Pass/Pass.h>
+
+namespace mlir {
+
+class RewritePatternSet;
+
+} // namespace mlir
+
+namespace mqt::ir::dyn {
+
+#define GEN_PASS_DECL
+#include "mlir/Dialect/MQTDyn/Transforms/Passes.h.inc" // IWYU pragma: export
+
+void populateFoldExtractQubitPatterns(mlir::RewritePatternSet& patterns);
```

Comment:
Could we improve the naming of this method slightly? Seems a bit confusing to me.
Maybe by adding a "Constant..." prefix.